server/data/data-jpa/src/test/java/org/apache/james/user/jpa/model/JPAUserTest.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.james.user.jpa.model;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class JPAUserTest {
+
+    private static final String RANDOM_PASSWORD = "baeMiqu7";
+
+    @Test
+    void hashPasswordShouldBeNoopWhenNone() {
+        //I doubt the expected result was the author intent
+        Assertions.assertThat(JPAUser.hashPassword(RANDOM_PASSWORD, "NONE")).isEqualTo("password");
```

Comment:
maybe you can open an issue for this one

Reply:
It's definitely a security issue but if you change that, you break existing installations.
What would you write in the issue ?

Reply:
NONE Hashing method should store password as clear text instead of using "password" string?

Reply:
https://issues.apache.org/jira/browse/MAILBOX-347